### PR TITLE
don't emit error when there no project.json files, hint to installation ...

### PR DESCRIPTION
**Pull request by jrieken**
...path if someone really want to DNX

### CODE REVIEW

File: src/OmniSharp/AspNet5/AspNet5Paths.cs [hunk 1]
```diff
@@ -27,23 +26,21 @@ public class AspNet5Paths
...
             _logger = loggerFactory.Create<AspNet5Paths>();
-            _emitter = emitter;
 
             RuntimePath = GetRuntimePath();
-            Dnx = FirstPath(RuntimePath, "dnx", "dnx.exe");
-            Dnu = FirstPath(RuntimePath, "dnu", "dnu.cmd");
-            Klr = FirstPath(RuntimePath, "klr", "klr.exe");
-            Kpm = FirstPath(RuntimePath, "kpm", "kpm.cmd");
-            K   = FirstPath(RuntimePath, "k", "k.cmd");
+            Dnx = FirstPath(RuntimePath.Value, "dnx", "dnx.exe");
```
Comment (davidfowl) [inline src/OmniSharp/AspNet5/AspNet5Paths.cs]: Can't this be null?

Comment (jrieken) [inline src/OmniSharp/AspNet5/AspNet5Paths.cs]: RuntimePath cannot, RuntimePath.Value can

Comment (davidfowl) [inline src/OmniSharp/AspNet5/AspNet5Paths.cs]: Ah we check for null

**Comment by MrTomWhite** [comment]
Do we need to leave this branch open? It wouldn't hurt to delete this and a few other old feature branches.

**Pull request merged**